mixitup/css/style.css

```diff
@@ -141,11 +141,14 @@ fieldset{
   margin: 0 5%;
 }
 
-.container .mix{
+#Container .mix{
```

Comment:
So I realized that the reason it is uppercase is that is what is required by the MixItUp plugin. Usually I don't upper case CSS selectors but in order for MixItUp to work you have to keep it uppercase (I tried switching it to lower case both in the CSS and HTML and the profiles didn't show at all).